modules/bbp-morphology/src/main/resources/schemas/bbp/morphology/annotatedslice/v0.1.0.json

```diff
@@ -0,0 +1,66 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0",
+    "{{base}}/contexts/nexus/prov/rec-prov-o-20130430/v0.1.0",
+    "{{base}}/contexts/bbp/core/entity/v0.1.0",
+    "{{base}}/contexts/nexus/core/standards/v0.1.0",
+    "{{base}}/contexts/bbp/neurosciencegraph/core/v0.1.0",
+    {
+      "bbpentsh": "{{base}}/schemas/bbp/core/entity/v0.1.0/shapes/",
+      "bbpqsh": "{{base}}/schemas/bbp/core/quantitativevalue/v0.1.0/shapes/",
+      "bbppsh": "{{base}}/schemas/bbp/core/person/v0.1.0/shapes/",
+      "this": "{{base}}/schemas/bbp/experiment/annotatedslice/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "owl:Ontology",
+  "imports": [
+    "{{base}}/schemas/bbp/core/entity/v0.1.0",
+    "{{base}}/schemas/bbp/core/quantitativevalue/v0.1.0"
+  ],
+  "shapes": 
+    [
+      {
+        "@id": "this:AnnotatedSliceShape",
+        "@type": "sh:NodeShape",
+        "description": "Annotated slice shape definition. An annotated slice should have an IRI as identifier",
+        "targetClass": "nsg:AnnotatedSlice",
+        "nodeKind": "sh:BlankNodeOrIRI",
+        "and": [
+          {
+            "node": "bbpentsh:EntityShape"
+          },
+          {
+            "property" : [ {
+              "path" : "nsg:AnnotationAngle",
+              "name" : "Annotation angle",
+              "description" : "Angle at which the slice is as defined during the annotation process",
+              "node": "bbpquantValue:QuantitativeValueShape",
+              "maxCount" : 1
+            }, {
+              "path" : "nsg:AnnotatorComment",
```

Comment:
I would suggest to use rdfs:comment as path value

Reply:
There are several "comment" fields. Some are for the agents to put a specific info on the activity, in this case the comment is only for the "Annotator" and specific to the "Annotation" activity. It is not a general comment.
In the "reconstruction" I also have a "ReconstructorComment".

Reply:
Okay. I still suggest to use a nsg:comment property for now and adopt later the following strategy when modelling comments:
 * if you care about provenance of a given comment (who comment, when, in which context ?) then it should be externalize as an entity with an identifier whom provenance can be asserted.
* sometimes one just want to put a text as a comment: in that case nsg:comment can have psd:string as datatype. 

To completely solve this, a model for storing comments is needed. I've created an [issue](https://github.com/BlueBrain/nexus-bbp-domains/issues/95) for that.

Reply:
Let keep "nsg:AnnotatorComment" until this [issue](https://github.com/BlueBrain/nexus-bbp-domains/issues/95) is solved.